x-pack/test/functional/services/transform/api.ts

```diff
@@ -110,6 +139,23 @@ export function TransformAPIProvider({ getService }: FtrProviderContext) {
       );
     },
 
+    async waitForTransformStateNotToBe(transformId: string, notExpectedState: TransformState) {
+      await retry.waitForWithTimeout(
+        `transform state not to be ${notExpectedState}`,
+        2 * 60 * 1000,
+        async () => {
+          const state = await this.getTransformState(transformId);
+          if (state !== notExpectedState) {
+            return true;
+          } else {
+            throw new Error(
+              `not expected transform state to be ${notExpectedState} but got ${state}`
```

Comment:
Super duper trivial: I think the phrasing here might get confusing ... maybe `expected transform state to not be ${notExpectedState} but got ${state}`? Not sure.

Reply:
Fixed in 2320fbc.